src/test/java/hw2/Ex1.java

```diff
@@ -33,18 +34,11 @@ public void SeleniumTestEx1() {
         assertEquals(driver.getTitle(), "Home Page");
 
         //6. Assert that there are 4 items on the header section are displayed and they have proper texts
-        WebElement homeBtn = driver.findElement(By.linkText("Home"));
-        WebElement contactFormBtn = driver.findElement(By.linkText("Contact form"));
-        WebElement serviceBtn = driver.findElement(By.linkText("Service"));
-        WebElement metalsAndColorsBtn = driver.findElement(By.linkText("Metals & Colors"));
-        assertTrue(homeBtn.isDisplayed());
-        assertTrue(contactFormBtn.isDisplayed());
-        assertTrue(serviceBtn.isDisplayed());
-        assertTrue(metalsAndColorsBtn.isDisplayed());
-        assertEquals(homeBtn.getText(), "Home");
-        assertEquals(contactFormBtn.getText(), "Contact form");
-        assertEquals(serviceBtn.getText(), "Service");
-        assertEquals(metalsAndColorsBtn.getText(), "Metals & Colors");
+        List<String> linkTextOfHeaderButtons = Arrays.asList("Home", "Contact form", "Service", "Metals & Colors");
```

Comment:
The pull request should not contain any other changes except changes according to the HW3